src/tools/EngineLauncher/packages.config

```diff
@@ -13,7 +13,7 @@
   <package id="Microsoft.VisualStudio.Shell.Interop.8.0" version="8.0.50727" targetFramework="net462" />
   <package id="Microsoft.VisualStudio.Shell.Interop.9.0" version="9.0.30729" targetFramework="net462" />
   <package id="Microsoft.VisualStudio.Utilities" version="15.0.26201" targetFramework="net461" />
```

Comment:
Should we also update:
1. [The sample's source.extension.vsixmanifest](https://github.com/microsoft/VSDebugAdapterHost/blob/34c3bf4a643c48f0d351c4843869b5a825bf0319/src/sample/SampleDebugAdapter.VSIX/source.extension.vsixmanifest#L9)
2. The engine launchers source.extension.vsixmanifest
3. These reference assemblies to point at the Dev17 versions

Reply:
I fixed the .vsixmanifests to make the VS versions open-ended and update the Newtonsoft reference.  I'm leaving the reference assemblies as-is so the samples still work in older VS (tested 2019 and 2022).